src/SFA.DAS.ApprenticeFeedback.Jobs/Infrastructure/Api/Requests/GenerateEmailTransactionsRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿using SFA.DAS.ApprenticeCommitments.Messages.Events;
+using System;
+
+namespace SFA.DAS.ApprenticeFeedback.Jobs.Infrastructure.Api.Requests
+{
+    public class GenerateEmailTransactionsRequest
```

Comment:
Not sure you need the request if the Post just has an empty body anyway? can you get away without it?

Reply:
Removed